crates/sui-network/src/discovery/mod.rs

```diff
@@ -59,6 +59,11 @@ pub struct NodeInfo {
     ///
     /// This is used to determine which of two NodeInfo's from the same PeerId should be retained.
     pub timestamp_ms: u64,
+
+    /// Access Type of this node.
+    /// If the node is Public, anyone could access it.
```

Comment:
optional/personal pref -  I'd keep this documented only on `AccessType` definintion so mutiple comments don't get out of sync